drivers/leds/leds-bd65b60.c

```diff
@@ -0,0 +1,318 @@
+	if (of_property_read_bool(np, "rohm,led2-used"))
+		pdata->led_sel |= BD65B60_LED2SEL;
+
+	pdata->no_reset = of_property_read_bool(np, "rohm,no-reset");
+
+	pdata->default_on = of_property_read_bool(np, "rohm,default-on");
+	pdata->init_level = BD65B60_DEFAULT_BRIGHTNESS;
+	if (pdata->default_on)
+		of_property_read_u32(np, "rohm,init-level", &pdata->init_level);
+
+	pdata->ovp_val = BD65B60_DEFAULT_OVP_VAL;
+	of_property_read_u32(np, "rohm,ovp-val", &pdata->ovp_val);
+
+	return 0;
+}
+
+static const struct of_device_id of_bd65b60_leds_match[] = {
+	{
+		.compatible = "rohm,bd65b60",
+	},
+	{ /* sentinel */ },
+};
+
+#else
+static int bd65b60_dt_init(struct i2c_client *client,
+			   struct bd65b60_platform_data *pdata)
+{
+	return ERR_PTR(-ENODEV);
+}
+
+static const struct of_device_id of_bd65b60_leds_match;
+
+#endif
+
+static int bd65b60_probe(struct i2c_client *client,
+			 const struct i2c_device_id *id)
+{
+	struct device_node *np = client->dev.of_node;
+	struct bd65b60_platform_data *pdata = dev_get_platdata(&client->dev);
+	struct bd65b60_chip *pchip;
+	struct led_init_data init_data = {};
+	int rc;
+
+	if (!i2c_check_functionality(client->adapter, I2C_FUNC_I2C)) {
+		dev_err(&client->dev, "fail : i2c functionality check");
+		return -EOPNOTSUPP;
+	}
+
+	pchip = devm_kzalloc(&client->dev, sizeof(struct bd65b60_chip),
+			     GFP_KERNEL);
+	if (!pchip)
+		return -ENOMEM;
+
+	if (!pdata) {
+		pdata = devm_kzalloc(&client->dev,
+				     sizeof(struct bd65b60_platform_data),
+				     GFP_KERNEL);
+		if (!pdata)
+			return -ENOMEM;
+
+		rc = bd65b60_dt_init(client, pdata);
+		if (rc)
+			return rc;
+	}
+
+	i2c_set_clientdata(client, pchip);
+	pchip->pdata = pdata;
+	pchip->dev = &client->dev;
+
+	pchip->regmap = devm_regmap_init_i2c(client, &bd65b60_regmap);
+	if (IS_ERR(pchip->regmap)) {
+		rc = PTR_ERR(pchip->regmap);
+		dev_err(&client->dev, "fail : allocate reg. map: %d", rc);
+		return rc;
+	}
+
+	/* chip initialize */
+	rc = bd65b60_chip_init(client);
+	if (rc < 0) {
+		dev_err(&client->dev, "fail : init chip");
+		return rc;
+	}
+
+	/* led classdev register */
+	pchip->cdev.brightness_set = bd65b60_led_set;
+	pchip->cdev.brightness_get = bd65b60_led_get;
+	pchip->cdev.max_brightness = BD65B60_MAX_BRIGHTNESS;
+	pchip->cdev.name = pdata->name;
+	pchip->cdev.default_trigger = pdata->trigger;
+	INIT_WORK(&pchip->ledwork, bd65b60_brightness_set);
+
+	init_data.fwnode = &np->fwnode;
+	rc = devm_led_classdev_register_ext(&client->dev, &pchip->cdev,
+					    &init_data);
+	if (rc) {
+		dev_err(&client->dev, "unable to register led rc=%d", rc);
+		return rc;
+	}
+
+	if (pdata->default_on)
+		bd65b60_led_set(&pchip->cdev, pdata->init_level);
+
+	return 0;
+}
+
+static void bd65b60_remove(struct i2c_client *client)
+{
+	struct bd65b60_chip *pchip = i2c_get_clientdata(client);
+
+	bd65b60_write(pchip, REG_PON, 0);
+	led_classdev_unregister(&pchip->cdev);
+}
+
+static const struct i2c_device_id bd65b60_id[] = { { BD65B60_NAME, 0 }, {} };
```

Comment:
```suggestion
static const struct i2c_device_id bd65b60_id[] = {
	{ BD65B60_NAME, 0 },
	{},
};
```

and maybe just put the string from the define here if nothing else uses it